netbout-web/src/test/java/com/netbout/rest/TkAppITCase.java

```diff
@@ -57,6 +58,18 @@
         Manifests.read("Netbout-Documentation");
 
     /**
+     * Xpath to the 'page not found' notice.
+     */
+    private static final String MISSING =
```

Comment:
@original-brownbear in another ticked I asked for constant just to be able to attach Javadoc, but this was very specific case